driver-core/src/main/com/mongodb/operation/ChangeStreamOperation.java

```diff
@@ -300,11 +334,21 @@ public BsonArray create(final ConnectionDescription description, final SessionCo
                     changeStream.append("allChangesForCluster", BsonBoolean.TRUE);
                 }
 
+                boolean hasResumeSetting = false;
```

Comment:
According to spec the server should throw the error if multiple resume settings are set.